tests/testcases/issue-72.nix

```diff
@@ -0,0 +1,5 @@
+{
+  foo = ''
+    bar
```

Comment:
this should probably be made more complicated, like:

```nix
{
  foo = ''
    bar
      baz
    tux
  '';
}
```

Reply:
Added a few lines more :)